### ISSUE TITLE
Best deep learming model

### ISSUE THREAD

**Issue by yvkrishna** [igkishore/vrushak#3]
Improve the current model accuracy

**Comment by igkishore** [comment]
need more help in model accuracy

**Comment by yvkrishna** [comment]
Yeah the custom generated model gives only 80 percentage accurate results. Shifting to inception for getting better results
